src/components/facet-browser/helper.ts

```diff
@@ -100,4 +102,15 @@ export function getAllFilterPathsAsString(filterObject: {
   return allFilterPathsAsString;
 }
 
+export const getFirstMaterialTypeFromFilters = (
+  filters: Filter,
+  manifestations: Manifestation[]
+) => {
+  const materialTypeFilter = Object.keys(filters.materialTypes || {}).sort()[0];
```

Comment:
Please look into if we can do better here.

Filters is currently just a map of strings and `materialTypes` is a value what we know should contain data. If this name changes our code will stille run but return no types. We get an exported enum `FacetField` with the value `materialTypes`. We should be able to use that.

Also: Grabbing the first entry with `[0]` should be replaced with `_.head()` from Lodash.